clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -922,13 +922,9 @@ public void testCommitsFetchedDuringAssign() {
 
         // fetch offset for two topics
         Map<TopicPartition, Long> offsets = new HashMap<>();
-        offsets.put(tp0, offset1);
-        client.prepareResponseFrom(offsetResponse(offsets, Errors.NONE), coordinator);
-        assertEquals(offset1, consumer.committed(Collections.singleton(tp0)).get(tp0).offset());
-
-        offsets.remove(tp0);
         offsets.put(tp1, offset2);
```

Comment:
Could we add a comment above about why we only need to respond with `tp1, offset2`? Something about it's been cached in previous committed offset fetch.

Reply:
good suggestion!